phpBB/phpbb/template/twig/environment.php

```diff
@@ -75,7 +75,7 @@ public function __construct(\phpbb\config\config $phpbb_config, \phpbb\filesyste
 
 		$options = array_merge(array(
 			'cache'			=> (defined('IN_INSTALL')) ? false : $cache_path,
-			'debug'			=> false,
+			'debug'			=> true,
```

Comment:
Why is this changed?

Reply:
I think I changed this for some more debugging and then forgot to change it back. Will revert this line.